CHANGES

```diff
@@ -1,6 +1,21 @@
 Bleach changes
 ==============
 
+Version 3.2.0.dev0 (May 15th, 2020)
```

Comment:
I only set the date here if it's been released. Otherwise it gets confusing in the documentation as to what was released and what's in the master branch that hasn't been released, yet.

So for this, I'd use "In development" or something along those lines.